pyproject.toml

```diff
@@ -2,5 +2,8 @@
 # To get a reproducible wheel, wheel must be pinned to the same version as in
 # dls-python3, and setuptools must produce the same dist-info. Cap setuptools
 # to the last version that didn't add License-File to METADATA
-requires = ["setuptools<57", "wheel==0.33.1"]
+requires = ["setuptools>=45,<57", "setuptools_scm[toml]>=6.2", "wheel==0.33.1"]
 build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+write_to = "src/scanspec/_version.py"
```

Comment:
This is also in setup.py, can we omit one of them?

Reply:
Or can it go in setup.cfg?

Reply:
Is it still required if we put things in setup.cfg?

Reply:
No `python -m setuptools_scm` won't use the config from setup.cfg. But, it doesn't really matter unless we want to check the version parsing works with any additional config, such as as tag_regex to allow a `+dlsX` suffix. Even then, you can do `python setup.py --version` instead.